docs/source/intro/datasets.rst

```diff
@@ -224,15 +227,16 @@ compatibility with ASReview.
 
 - Supported: The data can be exported from the software and imported in ASReview using this extension.
 - Not supported: The exported data can not be imported in ASReview using this extension.
-- (empty): The data cannot be exported from the software using this extension.
+- Empty: The extension format cannot be exported from the search engine.
 
-\* When using Covidence it is possible to export articles in .ris formats for different citation managers,
+\* When using Covidence it is possible to export articles in ``.ris`` format for different citation managers,
```

Comment:
According to the experts https://developers.google.com/style/filenames?hl=en#file-type-names, it is preferred to say `RIS file format`. I did apply this rule in several places.